.github/workflows/test.yml

```diff
@@ -1,56 +1,52 @@
-# Source:
-# https://github.com/solana-labs/solana-accountsdb-plugin-postgres/blob/master/.github/workflows/test.yml
+name: Build plerkle plugin
 
 on:
+  workflow_dispatch:
   push:
-    branches:
-      - main
-  pull_request:
-    branches:
-      - main
```

Comment:
I think the trigger we like the best (from your previous PR to RPC Infra) is run on any push to main, and any pull_request.